ui/e2e_tests/playground.spec.ts

```diff
@@ -355,3 +355,44 @@ test("playground should work with tool config ID different from display name @cr
     page.getByRole("heading", { name: "Inference Error" }),
   ).toHaveCount(0);
 });
+
+test("playground should run evaluation after inference @credentials", async ({
```

Comment:
we don't actually need credentials here I believe because all the inferences should be cacheable. Try removing it and we can regenerate fixtures to confirm this .